modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceReplacementProducts.testcase

```diff
@@ -184,6 +184,109 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-12579. As a buyer, I can view the discontinued SKU shown in the product's detail replacement tab"
```

Comment:
As a buyer, I can view discontinued SKUs shown in the replacement tab of the product's details page.